src/popup/snooze-content.js

```diff
@@ -10,8 +10,10 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import SnoozePopup from '../lib/components/SnoozePopup';
 
-// HACK: Arbitrary value found by measuring menu panel popup width
-const NARROW_PANEL_MIN_WIDTH = 225;
+// HACK: Arbitrary breakpoint for styles below which to use "narrow" variant
+// The panel width is specified in Firefox in em units, so it can vary between
+// platforms. OS X is around 224px, Windows is around 248px.
```

Comment:
Hmm, I only have a 1080p monitor on my Windows machine. Is this breaking for you? Can you see what the console messages say? There should be "resize" messages with the detected pixel width of the panel